packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -32,8 +35,17 @@ export interface TimeRangePickerProps {
   value: TimeRange;
   timeZone?: TimeZone;
   fiscalYearStartMonth?: number;
+
+  // If you handle sync state between pickers yourself use this prop to pass the sync button component. Otherwise
+  // a default one will show automatically if sync is possible
```

Comment:
```suggestion
    /** If you handle sync state between pickers yourself use this prop to pass the sync button component. 
   * Otherwise, a default one will show automatically if sync is possible.
   */
```
Let's use the multi-line comment syntax, otherwise they won't be shown in storybook.